ChangeLog

```diff
@@ -2,6 +2,23 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+Enhancement
+   * Only check for necessary RSA structure fields in `mbedtls_rsa_private`. In
+     particular, don't require P,Q if neither CRT nor blinding are
+     used. Reported and fix proposed independently by satur9nine and sliai
+     on GitHub.
```

Comment:
Shouldn't we also mention that `MBEDTLS_RSA_ALT` is now an available option in `config.h`? And also add a "New deprecations" section stating that direct access to members of `rsa_context` is now deprecated?

Reply:
Done.